#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Set first_weekday to Monday for fr_FR
# DP: Related bugs: 
# DP: Dpatch author: Jeff Bailey
# DP: Patch author: Jeff Bailey
# DP: Upstream status: Not submitted
# DP: Status Details: Locales not being submitted upstream until we
#     switch to belochs
# DP: Date: 2005-09-01

PATCHLEVEL=1

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.
--- glibc-2.3.5/localedata/locales/fr_FR	2005-09-01 21:31:57.000000000 -0400
+++ glibc-2.3.5/localedata/locales/fr_FR	2005-09-01 21:34:24.000000000 -0400
@@ -92,6 +92,7 @@
 END LC_NUMERIC
 
 LC_TIME
+first_weekday 2
 abday   "<U0064><U0069><U006D>";"<U006C><U0075><U006E>";/
         "<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
         "<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
